Fix: Added toast message for "no skills' filter

SubhraPriyadarshinee opened the pull request:
### Description

- Found similar code for "location filter"
- Copy pasted the same code below that

Fixes #866 

### Type of Change:

- Code

### How Has This Been Tested?

- Open app -> Go to members page -> open filter -> type anything on "skills" box that has zero results -> now it will show a toast message 


### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

SubhraPriyadarshinee committed:
fix: Toast message for "no skills" filter

Found similar code for "location" filter.
Copy pasted the same code below that.

SubhraPriyadarshinee committed:
fix: Toast message for "no skills" filter

Found similar code for "location" filter.
Copy pasted the same code below that.

SubhraPriyadarshinee committed:
fix: Toast message for "no skills" filter

Found similar code for "location" filter.
Copy pasted the same code below that.

vj-codes reviewed (dismissed):
Thank you for your contribution @SubhraPriyadarshinee 🎉

vj-codes commented:
@SubhraPriyadarshinee please update the PR description to `Fixes #866 ` so that this PR gets linked with the issue

SubhraPriyadarshinee committed:
fix: Validation of Task Description text before posting it to backend

Checked if textview is empty or not.
If empty, a toast message is shown.
Else, task is added.

SubhraPriyadarshinee commented:
Hello, Can you please delete the last pull request for fix #879 . I was trying to make a PR but it accidentally got merged with this one.

SubhraPriyadarshinee commented:
> Hello, Can you please delete the last pull request for fix #879 . I was trying to make a PR but it accidentally got merged with this one.

@vj-codes @epicadk

vj-codes commented:
@SubhraPriyadarshinee I am not sure what do you mean by "delete PR", please elaborate:)

SubhraPriyadarshinee commented:
> @SubhraPriyadarshinee I am not sure what do you mean by "delete PR", please elaborate:)
Actually, I was making a different PR for a different issue but somehow it got merged with this one and is showing these errors.

vj-codes commented:
@SubhraPriyadarshinee can you try reverting the last commit?

vj-codes commented:
Closing this PR due to inactivity.
Thank you for your contribution @SubhraPriyadarshinee :)
